src/main/scala/org/ergoplatform/nodeView/wallet/FilteringOptions.scala

```diff
@@ -0,0 +1,6 @@
+package org.ergoplatform.nodeView.wallet
+
+/**
+  * Request to filter transactions by height or by confirmations
+  */
+case class FilteringOptions(minHeight: Int, maxHeight: Int, minConfNum: Int, maxConfNum: Int)
```

Comment:
Would you consider to use "enum" with `Range` instead? 
Something along the lines:
```scala
sealed trait WalletFiltering;

case class Height(range: Range) extends WalletFiltering;
case class ConfirmationsNum(range: Range) extends WalletFiltering;
```

Reply:
Sounds nice! Done!